bridge-sdk/bridge-clients/solana-bridge-client/src/solana_bridge_client.rs

```diff
@@ -613,7 +613,7 @@ impl SolanaBridgeClient {
 
         let mint_account = client.get_account(&token).await?;
 
-        let mint_data = Mint::unpack(&mint_account.data)
+        let mint_data = Mint::unpack(&mint_account.data[..82])
```

Comment:
Why 82? And can you please create a const for that

Reply:
Basic length of a mint account. 2022 standard can have data beyond that

Reply:
Fixed.